README.md

```diff
@@ -102,12 +112,23 @@ spec:
 | `LEGO_KUBE_API_URL` | n | `http://127.0.0.1:8080` | API server URL |
 | `LEGO_LOG_LEVEL` | n | `info` | Set log level (`debug|info|warn|error`) |
 
-
 ## Full deployment examples
 
 - [Nginx Ingress Controller](examples/nginx/)
 - [GCE Load Balancers](examples/gce/)
 
+## Troubleshooting
+
+When interacting with *kube-lego*, its a good idea to run with `LEGO_LOG_LEVEL=debug` for more verbose details.
+Additionally, be aware of the automatically created resources (see environment variables) when cleaning up or testing.
+
+Possible resources for *help*:
+
+* `#coreos` on freenode
```

Comment:
I agree with @djt-code code, that it should be fine directing them to get help in a general kubernetes channel. Some problems turn out to be a bit more general, than just an issue with kube-lego. (The concept behind ingress and its controllers is quite a large bite for kubernetes novices. Had to go that myself)